docs/contributing/Building, Debugging, and Testing on Windows.md

```diff
@@ -72,8 +72,8 @@ give it a try.
 The Rosyln solution is designed to support easy debugging via F5.  Several of our
 projects produce VSIX which deploy into Visual Studio during build.  The F5 operation
 will start a new Visual Studio instance using those VSIX which override our installed
-binaries.  This means trying out a change to the language, IDE or debugger is as
-simple as hitting F5.
+binaries.  This means trying out a change to the IDE or debugger is as
+simple as hitting F5. Note that changes to the compiler aren't recommended to be tested using Roslyn Deployment.
```

Comment:
Rather than saying that something is not recommended, I think it is better to describe what should be expected. For example to say that out of process build by default doesn't use privately built version of the compiler. Why I think it is better, because there is still at least one way to get msbuild using the privately built version of the compiler by modifying project file and pointing it to the compiler, then end-to end testing would work.